src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
I'm not sure what a `LoginHeader` with a title looks like, but could we possibly do something like below so we could easily add more elements to it?

```jsx
<LoginHeader>
  <Title>Welcome back, please log in to your account</Title>
</LoginHeader>
```